src/app/token-options/mixins/new-due-time-mixin.ts

```diff
@@ -19,7 +19,7 @@ export function NewDueTimeMixin<TBase extends Constructor<IGridViewDataSource>>(
         constructor(...args: any[]) {
             super(...args);
             this.registerDataPointSource(() => ({
-                colName: 'New Assignment/Quiz Lock At',
+                colName: 'Change Canvas Assignment/Quiz’s Available Until to',
```

Comment:
Please see comment on line 62 of `spend-for-assignment-resubmission-token-option-field-component-factory.ts`.